pixl_core/tests/conftest.py

```diff
@@ -53,9 +53,10 @@
 os.environ["ORTHANC_URL"] = "http://localhost:8043"
 os.environ["ORTHANC_USERNAME"] = "orthanc"
 os.environ["ORTHANC_PASSWORD"] = "orthanc"  # noqa: S105, hardcoded password
-os.environ["DICOM_ENDPOINT_NAME"] = "test"
 # Endpoint for DICOMWeb server as seen from within Orthanc
-os.environ["DICOM_ENDPOINT_URL"] = "http://localhost:8042/dicom-web/"
+os.environ["DICOMWEB_URL"] = "http://dicomweb-server:8042/dicom-web"
```

Comment:
Yeah that's what got me as well, though I think it's due to some poor naming / defining on my part. This particular environment variable is just used in the `MockDicomWebUploader` class to set the `endpoint_url` field, which in the real class would be fetched from the Azure keyvault. I.e. it's the URL to the DICOMweb server _from the Orthanc server's point of view_. So here it should be the URL to the server within the Docker compose network.

It might be less confusing if I hardcode this (and the other `DICOMWEB_*` envvars) directly in the `MockDicomWebUploader.__init__()` method, maybe with a comment? Because these environment variables are not used anywhere else in the code, unlike the `ORTHANC_*` ones.

Reply:
So much fun, I think that's probably alright, I'd add a comment here explaining what it means and that should work out alright

Reply:
Updated comment in 918ba87a1c3c967bb7feeaa4b07252a6b258e8fe